clib/memprof_coq.mli

```diff
@@ -0,0 +1,53 @@
+(* From memprof-limits *)
+val is_interrupted : unit -> bool
+
+module Masking : sig
+
+  val with_resource :
+    acquire:('a -> 'b) -> 'a -> scope:('b -> 'c) -> release:('b -> unit) -> 'c
+
+  val is_blocked : unit -> bool
```

Comment:
This is used to determine if we are under a masked context, similarly `assert_blocked` asserts we are. This is useful to ensure we never call critical code without masking, so a usual pattern here is to start critical code with `assert_blocked`